### PR TITLE
Scrum 55

### PR DESCRIPTION
**Pull request by DannyPinghero**
[SCRUM-55]

[SCRUM-55]: https://gamepals.atlassian.net/browse/SCRUM-55

### CODE REVIEW

**Comment by DannyPinghero** [comment]
I _think_ this is better than using a signal handler to catch SIGINT ?  This should fire unless the interpreter itself hits an unrecoverable error.
I wonder if there should be a cronjob also that periodically pings the servers to see if anyone died unrecoverably

**Comment by DannyPinghero** [comment]
Ah you know I think registering the handler makes more sense in the run method not the __init__

File: backend/server.py [hunk 1]
```diff
@@ -20,13 +22,26 @@ def __init__(self) -> None:
...
-        redis_pubsub_instance = self.redis.pubsub(ignore_subscribe_messages=True,)
+        atexit.register(self.shutdown_handler)
 
+        redis_pubsub_instance = self.redis.pubsub(ignore_subscribe_messages=True,)
         redis_pubsub_instance.subscribe(
             **{"websocket-IPC": self.websocket_ipc_handler, "flask-IPC": self.flask_ipc_handler,}
         )
         self._redis_pubsub_thread = redis_pubsub_instance.run_in_thread(sleep_time=0.5,)
 
+    async def _shutdown_helper(self, tasks: List[Task[None]]) -> None:
```
Comment (acaggiano) [inline backend/server.py:33]: Remove Type Annotations

Comment (DannyPinghero) [reply backend/server.py:33]: *figure out how to _properly_ annotate that!

Comment (DannyPinghero) [reply backend/server.py:33]: probably just with a comment

**Pull request merged**